components/EditUpdateForm.js

```diff
@@ -205,6 +207,33 @@ class EditUpdateForm extends React.Component {
               />
             </Container>
           </div>
+
+          <div className="row">
+            <div className="col large">
+              <Container mb={2} fontWeight="500" fontSize="1.6rem" lineHeight="1.7">
+                <Box as="span">
+                  <FormattedMessage
+                    id="update.makePublicOn.label"
+                    defaultMessage="Automatically make the update public on this date:"
+                  />
+                </Box>
+              </Container>
+              <StyledInputField htmlFor="makePublicOn">
+                {inputProps => (
+                  <StyledInput
+                    {...inputProps}
+                    type="date"
+                    value={update.makePublicOn ? moment(update.makePublicOn).format('YYYY-MM-DD') : ''}
```

Comment:
We should avoid using `moment` for a simple formatting like that (though you couldn't know, because we do the same in multiple places of the code). I can suggest adding the following helper in https://github.com/opencollective/opencollective-frontend/blob/51a305389081047fe1a7eab3feae3062ef587c2c/lib/utils.js:

```es6
import { padStart } from 'lodash'

/** 
 * Format a datetime object to an ISO date like `YYYY-MM-DD`
 */
export const toIsoDateStr = (date) => {
  const year = date.getFullYear();
  const month = date.getMonth() + 1;
  const day = date.getDate();
  return `${year}-${padStart(month.toString(), 2, '0')}-${padStart(day.toString(), 2, '0')}`;
}
```

Reply:
Added... I changed to use the UTC Date, because in my local machine the function was returning the past date, because my timezone is -3 hours.